app/scripts/views/settings/settings-file-view.js

```diff
@@ -143,6 +146,17 @@ const SettingsFileView = Backbone.View.extend({
             });
             return false;
         }
+
+        if (this.model.get('passwordChanged') && !this.model.get('passwordMatched')) {
+            Alerts.error({
```

Comment:
This should be displayed inline, next to the confirmation field

Reply:
What happen if user click save button if the password entered not match?